app/models/color_scheme.rb

```diff
@@ -572,6 +576,51 @@ def is_dark?
   def is_wcag?
     base_scheme_id == NAMES_TO_ID_MAP["WCAG"] || base_scheme_id == NAMES_TO_ID_MAP["WCAG Dark"]
   end
+
+  def diverge_from_remote
+    new_scheme = dup
+    new_scheme.colors = self.colors_hashes
+    new_scheme.via_wizard = false
+    new_scheme.user_selectable = false
+    new_scheme.base_scheme_id = nil
+    new_scheme.skip_publish = true
+    new_scheme.remote_copy = true
+
+    DistributedMutex.synchronize("color_scheme_fork_#{self.id}") do
```

Comment:
```suggestion
    DistributedMutex.synchronize("color_scheme_diverge_#{self.id}") do
```

Nit for consistency.